libs/sdk-ui-kit/package.json

```diff
@@ -44,8 +44,8 @@
         "test-ci": "JEST_JUNIT_OUTPUT=./ci/results/test-results.xml jest --ci --config jest.ci.js",
         "eslint": "eslint $npm_package_config_eslint",
         "eslint-ci": "mkdir -p ./ci/results && eslint -f checkstyle -o ci/results/eslint-results.xml $npm_package_config_eslint",
-        "prettier-check": "prettier --check '{src,stories,styles,__mocks__}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
-        "prettier-write": "prettier --write '{src,stories,styles,__mocks__}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
+        "prettier-check": "prettier --check '{src,styles,__mocks__}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
+        "prettier-write": "prettier --write '{src,styles,__mocks__}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
```

Comment:
Why no prettier for stories? I can already see some discrepancies with `'` and `"` in stories below. Also, I believe I have prettier set up for all `*js` files, so it would automatically format these after touching them. Another thing is that all UI libs that have stories, have the same prettier setup.

Reply:
Yep, I will format them as well. These are no config files but regular source code.